ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1861,22 +1861,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-sno
-- as: gcp-ipi-labels-tags-filestore-csi-fips-tp-amd-f28-destructive
-  cron: 32 16 7 * *
+- as: gcp-ipi-labels-tags-filestore-csi-fips-amd-f28-destructive
```

Comment:
Can you also help update `gcp-ipi-user-labels-tags-filestore-csi-tp-arm-f28` in openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml